enterprise/backend/src/metabase_enterprise/api/routes.clj

```diff
@@ -33,14 +34,17 @@
   (context
    "/ee" []
    (context
-    "/billing" []
-    billing/routes)
+    "/advanced-permissions" []
```

Comment:
Given that this reordering clouds both the git blame and the PR diff, I'd have broken it out to its own micro PR. Water under the bridge now though.

Reply:
We'll see who wins the CI race: https://github.com/metabase/metabase/pull/45215